crt_portal/cts_forms/templates/forms/report_single_questions.html

```diff
@@ -0,0 +1,18 @@
+{% extends "forms/report_base.html" %}
+{% block form_questions %}
+<fieldset class="usa-fieldset">
+
+  <div class="page-subtitle">{{ page_subtitle }}</div>
+  {% for field in wizard.form.visible_fields %}
+    <div class="usa-prose margin-bottom-2">
+      <h3 class="usa-prose">{{ field.label }}</h3>
```

Comment:
One a11y note here — a piece of feedback we got from a blind user was that `<h3>`s shouldn't be used in form labels because their semantic meaning is relation to section heading. I think I created a `label.big` style for this use case instead.